central/complianceoperator/v2/integration/datastore/datastore.go

```diff
@@ -43,7 +43,9 @@ func New(complianceIntegrationStorage pgStore.Store, searcher search.Searcher) D
 }
 
 // GetTestPostgresDataStore provides a datastore connected to postgres for testing purposes.
-func GetTestPostgresDataStore(_ *testing.T, pool postgres.DB, searcher search.Searcher) (DataStore, error) {
+func GetTestPostgresDataStore(_ *testing.T, pool postgres.DB) DataStore {
 	store := pgStore.New(pool)
-	return New(store, searcher), nil
+	indexer := pgStore.NewIndexer(pool)
```

Comment:
Need to add function to delete integration by cluster

Reply:
It is already there.  `RemoveComplianceIntegrationByCluster`